cattle/plugins/docker/compute.py

```diff
@@ -215,6 +215,17 @@ def _determine_state(self, container):
             # Unknown. Assume running and state should sync up eventually.
             return 'running'
 
+    def _get_host_labels(self):
+        try:
+            labels = self.host_info.host_labels()
+        except:
+            log.exception("Error getting host labels")
```

Comment:
I don't have knowledge about what kind of exceptions can occur here, so was wondering if it would be useful for the user to see them in case he can go and manually fix something in order to recover.